keps/prod-readiness/sig-scheduling/2249.yaml

```diff
@@ -3,3 +3,5 @@ alpha:
   approver: "@wojtek-t"
 beta:
   approver: "@wojtek-t"
+stable:
+  approver: "@wojtek-t"
```

Comment:
@ahg-g I'm generally fine with it - just two things:
1) please get SIG lgtm on it first (I know you're the lead, I just want someone else to also officially stamp it)
2) one of the GA requirements were benchmarks to show that it causes performance issues
 Can you briefly summarize them? [what scenarios you were testing and what were the results]
 [I mean summarizing them in this KEP, not in this comment thread.]

Reply:
sg, Aldo (now a TL) already lgtmed, I also added a summary of the benchmark tests we did in the latest commit.